services/tests/test_repository_service.py

```diff
@@ -2076,6 +2031,10 @@ def test_fetch_commit_yaml_and_possibly_store_commit_yaml_invalid_commit_yaml(
     mock_configuration.set_params({"site": {"comment": {"behavior": "new"}}})
     commit = CommitFactory.create(
         repository__owner__yaml={"coverage": {"precision": 2}},
+        # User needs to be less than PATCH_CENTRIC_DEFAULT_TIME_START
```

Comment:
why is this needed?

Reply:
Ah, it's a remnant from https://github.com/codecov/worker/pull/729 . Basically, the new Django models adds a default time of now (SQLAlchemy models will just have no default time for the owner object). If the owner is created _after_ the 'PATCH_CENTRIC_DEFAULT_TIME_START', the default owner YAML is actually different and messed up with the tests. So, setting the timestamp is to preserve the test functionality of using the pre-patch_centric yaml behaviour. This doesn't really do anything _bad_ right now, and will be useful later when I start creating the tables with Django later.